orc8r/cloud/helm/orc8r/charts/secrets/values.yaml

```diff
@@ -56,11 +56,11 @@ secret:
         prometheusQueryAddress: "http://orc8r-prometheus:9090"
 
         alertmanagerApiURL: "http://orc8r-alertmanager:9093/api/v2/alerts"
-        prometheusConfigServiceURL: "http://orc8r-config-manager:9100"
-        alertmanagerConfigServiceURL: "http://orc8r-config-manager:9101"
+        prometheusConfigServiceURL: "http://orc8r-prometheus-configurer:9100/v1"
```

Comment:
I noticed that these values were outdated so I'm updating them here to avoid chart churn